src/test/scala/uk/gov/hmrc/wco/dec/inventorylinking/movement/request/InventoryLinkingMovementRequestFromXmlSpec.scala

```diff
@@ -39,7 +39,7 @@ class InventoryLinkingMovementRequestFromXmlSpec extends WcoSpec with XmlBehavio
         val inputXML = MovementRequestSpecInputXMLProvider.provideTestXMLFor("TestNoValueForOptionalField")
         val actualMovementRequest = InventoryLinkingMovementRequest.fromXml(inputXML)
 
-        actualMovementRequest.shedOPID mustNot be(defined)
+        actualMovementRequest.shedOPID must be(Some(""))
```

Comment:
This test no longer makes sense as it seems to be checking the opposite of what the test says it's looking for. Is this different behaviour of the new lib version, needs investigating why behaviour has changed  :thinking: